scripts/signature.py

```diff
@@ -156,6 +184,48 @@ def bias_signatures(base_signatures, contender_signatures, tickets):
     return "", status
 
 
+def gh_bias_check(args):
+    """
+    Exports signatures for the given group(s) and checks them for changes compared to a baseline.
+    Saves the results to a file and sets a Github Actions Output for each group.
+    """
+    if not os.getenv("GITHUB_ACTIONS"):
+        print("This command is meant to be run in a Github Actions environment.")
+        return 1
+
+    # export signatures for each group
+    for group in args.group:
+        print(f"Exporting {group} signatures...")
+        export_args = parse_args(
+            [
+                "export",
+                f"--{group}",
+                os.path.join(args.signature_dir, group + args.contender_postfix),
+            ]
+        )
+        export(export_args)
+
+    # compare signatures for each group
+    for group in args.group:
+        print(f"Comparing {group} signatures...")
+        bias_args = parse_args(
+            [
+                "bias",
+                os.path.join(args.signature_dir, group + args.base_postfix),
+                os.path.join(args.signature_dir, group + args.contender_postfix),
+                os.path.join(args.signature_dir, group + args.output_postfix),
+                os.path.join(args.signature_dir, group + "_errors"),
+            ]
+        )
+        bias_status = bias(bias_args)
+        set_output(f"{group}_error", str(bias_status == 1).lower())
```

Comment:
nit: I think it will be better if set_output( , _) takes a boolean and convert it to string than have the lower happen here.

Reply:
nit: or anytype and converts it to lower..